src/sentry/web/api.py

```diff
@@ -236,7 +236,7 @@ def _dispatch(self, request, helper, project_id=None, origin=None,
                 # to `Access-Control-Allow-Origin` and we don't have another
                 # value to work with, so just allow '*' since they've gotten
                 # this far.
-                response['Access-Control-Allow-Origin'] = '*'
+                response['Access-Control-Allow-Origin'] = 'null'
```

Comment:
Also, I feel like we can just safely return `*` here. I don't see any value in changing it. This change also makes the comment above it not make sense.